### PR TITLE
[FIX] table column actions icon

### LINKED ISSUE #554: Column filter going on second row
If Grid column content is shorter than column name, filter icon go on the next row
screen shot 2018-09-03 at 14 28 04 1

### PR DESCRIPTION
**Pull request by georgehristov**
fixes #554

a4d8720 is the actual fix
5b958f7 refactoring of variable names and using shorthand syntax for clarity

Print screen of the solution in action:
solution

### CODE REVIEW

**Comment by ibelar** [comment]
@georgehristov - Thanks for taking care of this. However, this ends up with the same problem as before, i.e. icon moving under the label name when there is no enough space. 

Screen Shot 2020-02-18 at 8 22 27 AM

It would be great if you could find a way to take care of this. For the rest, it looks very good!

**Comment by georgehristov** [comment]
@ibelar Thanks. The problem was that even if there is space the icon ended up below the caption.
Will look into it to keep them together always...

File: src/TableColumn/Generic.php [hunk 1]
```diff
@@ -294,74 +286,69 @@ public function getTag($position, $value, $attr = [])
...
-        } else {
-            $tag = $this->getTag(
-                'head',
-                [$captionHtmlTag],
-                $attr
-            );
-        }
-
-        return $tag;
+        return $this->getTag('head', [['div', compact('class'), $caption]], $attr);
```
Comment (georgehristov) [inline src/TableColumn/Generic.php]: I figured out how to do that but I get a problem with generating tags.

When generating the tag for the header all works well on atk4 side on the line linked to this comment.

The generated tag (var_dump) is:
```
<th data-column="accessed_at" id="atk_layoutview_fileaccesshistory_crud_view_table_generic_th" data-sort="accessed_at">
	<div class="atk-table-column-header">
		<div class="atk-table-dropdown">
			<i id="atk_layoutview_fileaccesshistory_crud_view_table_generic_ac" class="table-filter-off icon">
		</div>Accessed At</div>
</th>
```

Yet in the browser the column title is wrapped with a copy of the <i> tag?

```
<th data-column="accessed_at" id="atk_layoutview_fileaccesshistory_crud_view_table_generic_th" data-sort="-accessed_at">
	<div class="atk-table-column-header sorted ascending">
		<div class="atk-table-dropdown">
			<i id="atk_layoutview_fileaccesshistory_crud_view_table_generic_ac" class="table-filter-off icon"/>
		</div>
		<i id="atk_layoutview_fileaccesshistory_crud_view_table_generic_ac" class="table-filter-off icon">Accessed At</i>
	</div>
</th>
```

All works fine is text is "declared" first in the atk tag array

Any idea?

Comment (georgehristov) [reply src/TableColumn/Generic.php]: Found the issue - it is related to the fact that `<i>` tag is not closed when generated by atk

**Comment by georgehristov** [comment]
Now it should work properly

**Comment by georgehristov** [comment]
It was a typo causing both, please recheck

**Comment by ibelar** [comment]
Thanks, @georgehristov - Looks very good now!

**Comment by georgehristov** [comment]
@DarkSide666 thanks for noticing this. I introduced a fix

**Pull request merged**